Content.Shared/Jukebox/SharedJukeboxComponent.cs

```diff
@@ -0,0 +1,128 @@
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Jukebox;
+
+[NetworkedComponent, RegisterComponent]
+[Access(typeof(SharedJukeboxSystem))]
+public partial class JukeboxComponent : Component
+{
+    [DataField("playing")]
+    public bool Playing { get; set; }
+
+    [DataField("selectedSongId")]
+    public int SelectedSongID { get; set; }
+
+    [ViewVariables]
+    public MusicListDefinition? SelectedSong { get; set; }
+
+    [DataField("songTime")]
+    public float SongTime { get; set; }
+
+    [DataField("songStartTime")]
+    public float SongStartTime { get; set; }
+
+    [ViewVariables]
+    public IPlayingAudioStream? AudioStream { get; set; }
```

Comment:
unnecesary getters and setters, also there seems to be some data duplication since you should be able to get the SelectedSongID and time from SelectedSong, right?